app/controllers/home_controller.rb

```diff
@@ -157,4 +159,9 @@ def activity
     ]
     response
   end
+
+  def trending
+    exclude_tids = @tag_subscriptions.pluck(:tid)
+    Tag.trending.where.not(tid: exclude_tids)
```

Comment:
If the trending tags are all subset of  `@tag_subscriptions` then the list will be empty and we will not show anything, right? 
I think we should handle this corner case by showing the next sets of trending tags. 
So I think better approach is to change `self.trending` method and add `where.not(tid: exclude_tids)` inside it.

@jywarren thoughts?

Reply:
Thanks for the review @Sagarpreet 😃 

Concerning the trending tags being a subset of subscribed tags,I don't think it returns an empty list because when I tested this locally, if the trending tags contain two subscribed tags, the two tags are replaced by other tags ... the trending method in the tag model returns a limit of 5 tags from a certain period....


I'll investigate further and share

Reply:
@Sagarpreet I retested my assumption, 
<img width="1131" alt="Screenshot 2021-01-18 at 16 39 56" src="https://user-images.githubusercontent.com/7622875/104922546-ecad6b80-59ab-11eb-8a80-8bf0b232b5d7.png">

I first assigned the `exlucde_tags` :tids with the `Tag.trending` method
I then rerun the `Tag.trending` method, excluding the :tids and it returned a list of the next set of trending items. It did not return an empty list...

Reply:
Awesome got it 💯